### PR TITLE
ENH: Update extension metadata

### PR DESCRIPTION
**Pull request by jcfr**
Consolidate extension metadata based on the corresponding s4ext file organized
in the ExtensionsIndex repository.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -43,15 +43,17 @@ set(PRIMARY_PROJECT_NAME UKFTractography)
...
-set(EXTENSION_CATEGORY "Diffusion.Tractography")
-set(EXTENSION_CONTRIBUTORS "Yogesh Rathi, Stefan Lienhard, Yinpeng Li, Martin Styner, Ipek Oguz, Yundi Shi, Christian Baumgartner, Ryan Eckbo, Kent Williams, Hans Johnson, Peter Savadjiev, Carl-Fredrik Westin")
+set(EXTENSION_HOMEPAGE "https://www.nitrc.org/projects/ukftractography/")
+set(EXTENSION_CATEGORY "Tractography")
+set(EXTENSION_CONTRIBUTORS "Yogesh Rathi (yogesh@bwh.harvard.edu), Stefan Lienhard, Yinpeng Li, Martin Styner, Ipek Oguz, Yundi Shi, Christian Baumgartner, Ryan Eckbo, Hans Johnson, Kent Williams, Peter Savadjiev, Carl-Fredrik Westin.")
 set(EXTENSION_STATUS "Alpha")
 set(EXTENSION_DEPENDS "NA")
-set(EXTENSION_DESCRIPTION "This module traces fibers in a DWI Volume using the multiple tensor unscented Kalman Filter methology. ")
+set(EXTENSION_DESCRIPTION "This extension provides a module for tracing fibers in a DWI Volume using the multiple tensor unscented Kalman Filter methology.
+
```
Comment (tashrifbillah) [inline CMakeLists.txt:52]: Correct typos:
Volume --> volume (lower case)
methology --> methodology

**Comment by jcfr** [comment]
Now that the comments have been addressed, could this be approved & merged :pray:

**Comment by tashrifbillah** [comment]
Thank you Jean. I merged it.

**Comment by jcfr** [comment]
@tashrifbillah  Thanks for the follow-up :pray: 

Now that the changes have been reviewed and the checks pass, could you merge the pull request ?

**Comment by tashrifbillah** [comment]
> Thank you Jean. I merged it.

Sorry, just realized that I only commented but did not merge it. We have been in a hackathon ever since.

**Comment by jcfr** [comment]
No worry, and thanks again for taking the time to follow-up :pray: :rocket: 

Hope the hackathon is going well :crossed_fingers:

**Pull request merged**